server/src/main/java/org/opensearch/cluster/service/ClusterManagerTaskThrottler.java

```diff
@@ -185,6 +189,27 @@ public void onBeginSubmit(List<? extends TaskBatcher.BatchedTask> tasks) {
         });
     }
 
+    /**
+     * If throttling thresholds are set via static setting, it will update the threshold map.
+     * It may start throwing throttling exception to older nodes in cluster.
+     *
+     * On submission of first task, it validates if all nodes are of 2.5.0 or not and
+     * update startThrottling flag.
+     * Once the flag is set it will start throttling, and won't perform check for next tasks.
+     */
+    private boolean checkForThrottling(Long threshold, Long count, int size) {
+        if (!startThrottling.get()) {
+            if (minNodeVersionSupplier.get().compareTo(Version.V_2_5_0) >= 0) {
+                startThrottling.compareAndSet(false, true);
+                logger.info("Starting cluster manager throttling as all nodes are higher than or equal to 2.5.0");
+            } else {
+                logger.info("Skipping cluster manager throttling as node older than 2.5.0 is present in cluster");
```

Comment:
minor -> as at least one node < 2.5.0 is present in cluster

Reply:
Ack